usmqe/api/tendrlapi/cephapi.py

```diff
@@ -11,10 +11,11 @@
 class TendrlApiCeph(TendrlApi):
     """ Ceph methods for Tendrl REST API.
     """
-    def import_ceph_cluster(self, nodes):
+    def import_cluster(self, nodes, asserts_in=None):
```

Comment:
That was my original idea, not to create a new method but change the inherited one. I didn't noticed it's not the same method, when I reviewed your PR.